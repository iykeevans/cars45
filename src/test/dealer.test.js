import { render } from "./test-utils";

import Dealer from "../pages/dealer";

describe("About Page", () => {
  test("About page matches snapshot", () => {
    const { asFragment } = render(<Dealer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
