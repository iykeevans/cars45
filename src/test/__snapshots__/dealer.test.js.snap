// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page About page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header-one"
  >
    <div
      class="container"
    >
      <div
        class="row justify-content-md-end"
      >
        <div
          class="col-10 col-lg-4 phone-box"
        >
          <div
            class="top-phone"
          >
            <img
              alt="phone"
              src="/assets/icons/phone.svg"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
            <div
              class="divider"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
          </div>
        </div>
        <div
          class="col-1 col-md-1 profile"
        >
          <div
            class="btn-group"
          >
            <button
              aria-expanded="false"
              aria-haspopup="true"
              class="btn btn-outline-secondary dropdown-toggle"
              data-toggle="dropdown"
              type="button"
            >
              <img
                alt="profile"
                src="https://storage.googleapis.com/cars45-web-bucket/profile-icon.svg"
              />
            </button>
            <div
              class="dropdown-menu"
            >
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Sign In
              </button>
              <div
                class="dropdown-divider"
              />
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Register
              </button>
            </div>
          </div>
        </div>
      </div>
      <nav
        class="navbar navbar-expand-lg navbar-light bg-light"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="logo"
            class="logo"
            src="https://storage.googleapis.com/cars45-web-bucket/logo.svg"
          />
        </a>
        <button
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-target="#navbarNavDropdown"
          data-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarNavDropdown"
        >
          <ul
            class="navbar-nav"
          >
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/brandnew"
              >
                Brand New Cars 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary used"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Used Cars
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu used-menu"
              >
                <a
                  class="dropdown-item"
                  href="/buy"
                >
                  Buy
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/sell"
                >
                  Sell
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/swap"
                >
                  Swap
                </a>
              </div>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary inspection"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Inspection Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu inspect"
              >
                <a
                  class="dropdown-item"
                  href="/ride-hailing"
                >
                  Ride Hailing 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/private-individual"
                >
                  Private/Individual 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/premium-inspection"
                >
                  Premium Inspection 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/valuation"
                >
                  Valuation 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/due-dilligence"
                >
                  Due Dilligence 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/all-in-one"
                >
                  All-in-one 
                </a>
              </div>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/loan"
              >
                Car Loan 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/service"
              >
                Service & Repair 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/franchise"
              >
                Become a Franchise 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/corporate"
              >
                Corporate Service 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/import"
              >
                Import Cars 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/referral"
              >
                Referral Program 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Other Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu"
              >
                <a
                  class="dropdown-item"
                  href="/about"
                >
                  About Us 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/check"
                >
                  Background Check 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/dealer"
                >
                  Become A Dealer 
                </a>
              </div>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </div>
  <div
    class="dealer"
  >
    <div
      class="hero"
      style="background: url(https://storage.googleapis.com/cars45-web-bucket/become-a-dealer.webp) no-repeat center;"
    />
    <div
      class="text-area container pb-5  "
    >
      <div
        class="heading mb-4 text-center"
      >
        Become a Dealer
      </div>
      <div
        class="text"
      >
        Get access to our inventory today!
      </div>
      <div
        class="container quality pt-5 pz-0"
      >
        <div
          class="top px-lg-5 mx-4 mx-lg-5"
        >
          <div
            class="heading"
          >
            Quality cars for the best prices
          </div>
          <div
            class="row sign-up"
          >
            <div
              class="sign-up-left col-lg-8"
            >
              <div
                class="sign-up-left-heading"
              />
              <ul
                class="mt-4"
              >
                <li
                  class="d-flex align-items-center"
                >
                  <div>
                    <img
                      alt="check mark"
                      class="mr-3"
                      src="https://storage.googleapis.com/cars45-web-bucket/checkmark-circle.svg"
                    />
                  </div>
                   All car inspected and document checked
                </li>
                <li
                  class="d-flex align-items-center"
                >
                  <div>
                    <img
                      alt="check mark"
                      class="mr-3"
                      src="https://storage.googleapis.com/cars45-web-bucket/checkmark-circle.svg"
                    />
                  </div>
                   No-fuss actions-bid or buy with one click
                </li>
                <li
                  class="d-flex align-items-center"
                >
                  <div>
                    <img
                      alt="check mark"
                      class="mr-3"
                      src="https://storage.googleapis.com/cars45-web-bucket/checkmark-circle.svg"
                    />
                  </div>
                   Full support team available to answer your questions
                </li>
              </ul>
              .
            </div>
            <div
              class="sign-up-right col-12 col-lg-4 d-flex flex-column "
            >
              <img
                alt="mobile phone"
                class="mx-auto"
                src="https://storage.googleapis.com/cars45-web-bucket/phone.svg"
              />
              <div
                class="d-flex flex-column flex-lg-row mx-auto mb-2 mt-2"
              >
                <a
                  href="https://play.google.com/store/apps"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <img
                    alt="google play store"
                    class="mr-3 mb-4 mb-lg-0 mt-3 mt-lg-0"
                    src="https://storage.googleapis.com/cars45-web-bucket/google-play.svg"
                  />
                </a>
                <a
                  href="https://www.apple.com/app-store/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <img
                    alt="app store"
                    class="mr-3"
                    src="https://storage.googleapis.com/cars45-web-bucket/app-store.svg"
                  />
                </a>
              </div>
            </div>
          </div>
          <button
            class="mx-auto btn d-flex sign-up-btn mt-5"
          >
            sign up now
          </button>
        </div>
        <div
          class="how-work p-5"
        >
          <div
            class="heading text-center"
          >
            How it works
          </div>
          <div
            class="row justify-content-between mt-5 "
          >
            <div
              class="col-lg-5 d-flex flex-column align-items-center justify-content-center"
            >
              <div>
                <img
                  alt="sign up"
                  class="mr-3"
                  src="https://storage.googleapis.com/cars45-web-bucket/sign-up.svg"
                />
              </div>
              <div
                class="title text-center my-3"
              >
                Sign up
              </div>
              <div
                class="content text-center"
              >
                Register your company! We verify every dealer to ensure our
    auctions stay secure and hassle-free.
              </div>
            </div>
            <div
              class="col-lg-5 d-flex flex-column align-items-center justify-content-center"
            >
              <div>
                <img
                  alt="download the app"
                  class="mr-3"
                  src="https://storage.googleapis.com/cars45-web-bucket/download.svg"
                />
              </div>
              <div
                class="title text-center my-3"
              >
                Download the app
              </div>
              <div
                class="content text-center"
              >
                Have constant access to all auctions via our mobile app.
              </div>
            </div>
            <div
              class="col-lg-5 d-flex flex-column align-items-center justify-content-center"
            >
              <div>
                <img
                  alt="browse and bid"
                  class="mr-3"
                  src="https://storage.googleapis.com/cars45-web-bucket/browses.svg"
                />
              </div>
              <div
                class="title text-center my-3"
              >
                Browse and Bid
              </div>
              <div
                class="content text-center"
              >
                Browse our auction lists, find the ones that interest you and place a bid
              </div>
            </div>
            <div
              class="col-lg-5 d-flex flex-column align-items-center justify-content-center"
            >
              <div>
                <img
                  alt="done"
                  class="mr-3"
                  src="https://storage.googleapis.com/cars45-web-bucket/done.svg"
                />
              </div>
              <div
                class="title text-center my-3"
              >
                Done
              </div>
              <div
                class="content text-center"
              >
                If you win an auction, the car is yours!
              </div>
            </div>
          </div>
        </div>
        <form
          class=" col-lg-7 d-flex flex-column p-5 "
        >
          <a
            href="https://dealer.cars45.com/signup/"
          >
            Get Started
          </a>
        </form>
      </div>
    </div>
    <div
      class="verification "
    >
      <div
        class="heading"
      >
        Our verification locations
      </div>
      <p
        class="text-center mt-3"
      >
        Verification can be performed at the following locations
      </p>
      <div
        class=" col-10 mx-auto text-white "
      >
        <div
          class=" d-none d-lg-flex  justify-content-between border-bottom font-weight-bold pb-3 mt-5 h5"
        >
          <div
            class="col-3"
          >
            Lagos
          </div>
          <div
            class="col-3"
          >
            Port harcourt
          </div>
          <div
            class="col-3"
          >
            Abuja
          </div>
        </div>
      </div>
      <div
        class=" col-10 mx-auto text-white"
      >
        <div
          class=" d-flex flex-column flex-lg-row justify-content-between  mt-3 pb-3"
        >
          <div
            class=" col-12 col-lg-3 mt-5 mt-lg-0"
          >
            <p
              class="h5 font-weight-bold d-lg-none"
            >
              Lagos
            </p>
            Alausa Shopping Mall Suite 6, Block B, Second Floor, Obafemi
      Awolowo way, Alausa, Ikeja
          </div>
          <div
            class=" col-12 col-lg-3 mt-5 mt-lg-0"
          >
            <p
              class="h5 font-weight-bold d-lg-none"
            >
              Port harcourt
            </p>
            Port harcourt&lt;/p&gt;
        DRC Cars45 Dealer Resource Center. 57 Uyo Street off Stadium Rd,
        it’s a red plaza Gracious Plaza opposite Save A Life Hospital.
        Phone: 09087026161
          </div>
          <div
            class=" col-12 col-lg-3 mt-5 mt-lg-0"
          >
            <p
              class="h5 font-weight-bold d-lg-none"
            >
              Abuja
            </p>
            ALIBRO ATRIUM SUIT S-6 2ND FLOOR. NO 32 A EKUKINAM STREET UTAKO
        DISTRICT. Landmark: Close to ABC Transport
          </div>
        </div>
        <div
          class="text-center font-weight-bold h5 mt-3"
        >
          Do you have questions?
        </div>
        <p
          class="text-center mt-3 "
        >
          Our support team is happy to hear from you! 
        </p>
        <p
          class="text-center d-flex justify-content-center mt-3 pb-5 mb-0"
        >
           
          <span>
            <img
              alt="app store"
              class="mr-3"
              src="https://storage.googleapis.com/cars45-web-bucket/telephone.svg"
            />
          </span>
          +234 813 984 0160
        </p>
      </div>
    </div>
    <div />
  </div>
  <div>
    <div
      class="footer-one border-top"
    >
      <div
        class="footer-menu"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-9 col-md-7"
            >
              <div
                class="row"
              >
                <div
                  class="col col-md-2 d-none d-md-block"
                >
                  <a
                    href="/about"
                  >
                    About Us
                  </a>
                </div>
                <div
                  class=" col col-md-2"
                >
                  <a
                    href="/"
                  >
                    Terms Of Use
                  </a>
                </div>
                <div
                  class="col col-md-2"
                >
                  <a
                    href="/"
                  >
                    Privacy Policy
                  </a>
                </div>
                <div
                  class="col col-md-1"
                >
                  <a
                    href="/"
                  >
                    FAQs
                  </a>
                </div>
                <div
                  class="col col-md-2"
                >
                  <a
                    href="/loan"
                  >
                    Financing
                  </a>
                </div>
                <div
                  class="col col-md-3 d-none d-md-block"
                >
                  <a
                    href="/franchise"
                  >
                    Become A Franchise
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-1 col-md-5"
            >
              <div
                class="row"
              >
                <div
                  class="col-md-6 text-center"
                >
                  <a
                    href="mailto:help@car45.com"
                  >
                    help@car45.com
                  </a>
                </div>
                <div
                  class="col-md-6 text-right d-none d-md-block"
                >
                  <div
                    class="socials"
                  >
                    <a
                      href=" http://facebook.com/cars45.NG"
                      target="_blank"
                    >
                      <img
                        alt="facebook"
                        class="facebook"
                        src="https://storage.googleapis.com/cars45-web-bucket/fb.svg"
                      />
                    </a>
                    <a
                      href="https://www.instagram.com/cars45ng/"
                      target="_blank"
                    >
                      <img
                        alt="instagram"
                        src="https://storage.googleapis.com/cars45-web-bucket/insta.svg"
                      />
                    </a>
                    <a
                      href="https://twitter.com/cars45ng"
                      target="_blank"
                    >
                      <img
                        alt="twitter"
                        src="https://storage.googleapis.com/cars45-web-bucket/tw.svg"
                      />
                    </a>
                    <a
                      href="/youtube.com"
                      target="_blank"
                    >
                       
                      <img
                        alt="youtube"
                        src="https://storage.googleapis.com/cars45-web-bucket/yt.svg"
                      />
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="footer-section-one"
      >
        <div
          class="container"
        >
          <div
            class="row info"
          >
            <div
              class="col-md-7"
            >
              <p>
                Cars45 brand provides consumers with the opportunity to buy cars online and offline. Car dealers and end-users can now have access to a rich variety of cars that have been properly verified and checked, accompanied by a standard inspection report. What we are bringing to the market isn't just availability but access to cars with the right information such that consumers can make the right decision. Cars45 currently has its operations spread across Lekki, Ikeja, Ikorodu, Oshodi and Amuwo areas of Lagos, Ibadan, Kano, Port-Harcourt, and Abuja.
              </p>
            </div>
            <div
              class="col-md-5 subscribe"
            >
              <p>
                SUBSCRIBE TO GET NOTIFIED
                <br />
                 ON GREAT OFFERS ON CARS45
              </p>
              <form
                class="form-inline"
              >
                <div
                  class="form-group mr-4 mb-2"
                >
                  <input
                    class="form-control"
                    id="email"
                    name="email"
                    placeholder="Your email address"
                    type="email"
                  />
                </div>
                <button
                  class="btn btn-primary mb-2 teal-button"
                  type="submit"
                >
                  SUBSCRIBE
                </button>
              </form>
            </div>
          </div>
          <div
            class="row ssl"
          >
            <div
              class="col-md-6"
            >
              <p>
                SSL Protection | Buy with confidence
              </p>
              <p>
                buy.cars45.com has been security-scanned and is 100% protected by the Secure Socket Layer certificate. All files and data sent over this site are fully encrypted from third-party access.
              </p>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="socials"
              >
                <a
                  href="/facebook.com"
                >
                  <img
                    alt="facebook"
                    class="facebook"
                    src="https://storage.googleapis.com/cars45-web-bucket/facebook.svg"
                  />
                </a>
                <a
                  href="/instagram.com"
                >
                  <img
                    alt="instagram"
                    src="https://storage.googleapis.com/cars45-web-bucket/instagram.svg"
                  />
                </a>
                <a
                  href="/twitter.com"
                >
                  <img
                    alt="twitter"
                    src="https://storage.googleapis.com/cars45-web-bucket/twitter.svg"
                  />
                </a>
                <a
                  href="/youtube.com"
                >
                   
                  <img
                    alt="youtube"
                    src="https://storage.googleapis.com/cars45-web-bucket/youtube.svg"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="footer-section-two"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <ul>
                <li>
                  <a
                    href="/"
                  >
                    Terms & Conditions
                  </a>
                </li>
                <li>
                  <a
                    href="/"
                  >
                    Privacy
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="col-md-6 text-center text-md-right"
            >
              <p>
                2021Cars45.com. ©All rights reserved
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
